docs/build-for-developers/cli-usage.md

```diff
@@ -173,8 +177,8 @@ openfn repo clean
 
 ### Run a workflow
 
-<details>
-  <summary>A workflow has a structure like this</summary>
+<details open>
```

Comment:
hmm. I take your point but I'd probably rather hide the workflow json, it's a bit big. I can't even remember why we're including it.

Can we instead rename the summary text to `Click to see a workflow's JSON structure` or something?